src/dosext/mfs/lfn.c

```diff
@@ -1328,6 +1328,11 @@ int mfs_lfn(void)
 {
 	int carry, ret;
 
+	if (!mach_fs_enabled) {
+		CARRY;
+		return 0;
+	}
+
```

Comment:
Could you please point me where does
this happen? I thought we do not implement
QUALIFY_FILENAME at all.

With vfat/int13 backend there will be no
QUALIFY_FILENAME involved. lfn will
just have to figure out SFN for the requested
lfn, and pass that down to DOS's open(),
which will call int13 from then.
And this is actually what lfn.c is doing even
now! If it calls into mfs.c too much, then
this should probably be cleaned up. So
what does it do with mfs?